cabal-install/tests/IntegrationTests2.hs

```diff
@@ -1971,8 +1982,8 @@ testNixFlags = do
 -- Tests whether config options are commented or not
 testConfigOptionComments :: Assertion
 testConfigOptionComments = do
-  _ <- createDefaultConfigFile verbosity [] (basedir </> "config/default-config")
-  defaultConfigFile <- readFile (basedir </> "config/default-config")
+  _ <- createDefaultConfigFile verbosity [] (basedir </> "config" </> "default-config")
+  defaultConfigFile <- readFile (basedir </> "config" </> "default-config")
```

Comment:
This is just a cosmetic change, and the config was crafted even before this patch, is that right? Why that was not working? Was the config that's being discussed in these lines not picked up by Cabal? Did it not exist? A little explanation would be great in the commit message, PR, or both.

Reply:
Also, the only change here is using `</>` instead of a hard-coded `/`. It _might_ only be cosmetic, but I suspect it's actually a correctness change that happens not to matter currently.

Reply:
Also, this kind of discussion is why I solicited reviews in draft mode.